test/test_suite.py

```diff
@@ -52,6 +52,7 @@
 from sqlalchemy.testing.suite.test_cte import *  # noqa: F401, F403
 from sqlalchemy.testing.suite.test_ddl import *  # noqa: F401, F403
 from sqlalchemy.testing.suite.test_dialect import *  # noqa: F401, F403
+from sqlalchemy.testing.suite.test_insert import *  # noqa: F401, F403
```

Comment:
Adding `insert` tests group into workflow, as all of the tests are processed: most of them are skipped, as Spanner doesn't support `RETURNING`, default column values, primary key autoincrement and last inserted row id feature.